app/src/main/java/com/ucmobiledevelopment/freeways/ui/theme/Theme.kt

```diff
@@ -6,13 +6,13 @@ import androidx.compose.material.darkColors
 import androidx.compose.material.lightColors
 import androidx.compose.runtime.Composable
 
-private val DarkColorPalette = darkColors(
+private val darkColorPalette = darkColors(
     primary = Purple200,
     primaryVariant = Purple700,
     secondary = Teal200
 )
 
-private val LightColorPalette = lightColors(
+private val lightColorPalette = lightColors(
```

Comment:
Changed these to fit Kotlin naming conventions. Camel Case seems appropriate considering the data that these values hold.